MAINTAINERS.yml

```diff
@@ -421,6 +421,28 @@ Bluetooth Audio:
   tests:
     - bluetooth.audio
 
+Bluetooth classic:
+  status: maintained
+  maintainers:
+    - lylezhu2012
+  collaborators:
+    - jhedberg
+    - sjanc
+  files:
+    - subsys/bluetooth/host/a2dp*
+    - subsys/bluetooth/host/at.*
+    - subsys/bluetooth/host/avdtp*
+    - subsys/bluetooth/host/br.*
+    - subsys/bluetooth/host/hfp*
+    - subsys/bluetooth/host/l2cap_br*
+    - subsys/bluetooth/host/rfcomm*
+    - subsys/bluetooth/host/sdp*
+    - subsys/bluetooth/host/ssp*
+  labels:
+    - "area: Bluetooth Host"
```

Comment:
I think we need to add a new 'area: Bluetooth Classic' label. That way you can just add the label to issues and zephyrbot will automatically set the correct assignee.

Reply:
https://github.com/zephyrproject-rtos/zephyr/labels/area%3A%20Bluetooth%20Classic there you go